etc/profile.d/aliases.sh

```diff
@@ -10,7 +10,7 @@ xterm*)
 	# The following programs are known to require a Win32 Console
 	# for interactive usage, therefore let's launch them through winpty
 	# when run inside `mintty`.
-	for name in node ipython php php5 psql python2.7
+	for name in node ipython php php5 psql python2.7 python
```

Comment:
Makes sense.

However, this file is not _maintained_ in git-sdk-64, it is merely mirrored here. It originally comes from https://github.com/git-for-windows/build-extra/blob/HEAD/git-extra/aliases.sh and should be changed there (read: could you please open a PR in https://github.com/git-for-windows/build-extra/ instead?).

Oh, and could you please describe the reasoning behind this change a bit more verbosely? Please follow the example of the commit message of git-for-windows/build-extra@5825d46e403c8755557b2d552ce118150f9fd036. This will help tremendously when other developers (or yourself) stumble across this patch and wonder about the rationale behind it.

Thanks!